pages/arduino-info.mdx

```diff
@@ -0,0 +1,129 @@
+import Head from 'next/head'
+import Meta from '@hackclub/meta'
+
+<Meta
+  as={Head}
+  name="Summer of Making"
+  title="About your Arduino Starter Kit"
+/>
+
+<main className="container">
+
+# About your Arduino Starter Kit
+
+For starters, congrats on earning a kit! We're really excited to see what you make with yours!
+These kits are made possible by the gracious support of [Github](https://github.com) & [Arduino](https://arduino.cc).
+The aim of this document it to provide you with all the information you need about your grant.
+
+### When will my kit arrive?
+
+We will place your order within 48 hours of you submitting the form. You will receive a DM when Arduino ships it containing tracking info. It will arrive in anywhere from 3 days to 1 week if you are based in the US or Europe. International deliveries may take up to 2 and a half weeks. 
+
+### How can I get my own referral codes?
+
+To get your codes, you first need to make something. After making something share a photo of what you've built in your `#scrapbook`. Include a git repo link with photos + code and react with `:arduino-kit:`! You will be presented with further information then through a Slack Bot.
+
+### What is my kit?
+
+1 Projects Book (170 pages),
+
+1 Arduino Uno & USB Cable
+
+1 Breadboard 400 points & 70 Solid core jumper wires,
+
+Loads of assorted electronic components!
+
+Here is a [link to the product page for exact components.]()
+
+### Who can I contact for more questions?
+
+Please join `#hardware-party-questions` and ask a question there. A member of the Summer of Making team will assist you within 24 hours.
+
+### Happy hacking!
+
+![](https://media3.giphy.com/media/26BRHKYhKLd6PlwsM/source.gif)
+
+</main>
+
+<style jsx global>{`
+  .container {
+    max-width: 680px !important;
+    font-size: 20px !important;
+    line-height: 1.625;
+    margin: auto;
+    margin-top: 30px;
+    margin-bottom: 30px;
+  }
+
+  a {
+    color: var(--colors-orange);
+  }
+  a:hover,
+  a:focus {
+    text-underline-position: under;
+    text-decoration-style: wavy;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    line-height: 1.25;
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+  }
+
+  h1 {
+    font-size: 3rem;
+    margin-top: 0;
+  }
+  h2 {
+    font-size: 2rem;
+  }
+  h3 {
+    font-size: 1.5rem;
+  }
+
+  p,
+  ol,
+  ul,
+  pre,
+  blockquote {
+    margin-top: 1em;
+    margin-bottom: 1em;
+  }
+
+  small {
+    color: var(--colors-muted);
+  }
+
+  code,
+  pre {
+    font-family: 'SF Mono', 'Roboto Mono', 'Source Code Pro', Menlo, Consolas,
+      'Liberation Mono', monospace;
+    background-color: beige;
+    font-size: 87.5%;
+    color: var(--colors-purple);
+  }
+
+  @media (prefers-color-scheme: dark) {
```

Comment:
V optional but changing the page background/color here would be nice, you can get all the colors from the theme file

Reply:
I can play around with the theme and stuff but really I just wanted a quick and easy way to share a doc that looked nice

Reply:
https://github.com/hackclub/summer-site/pull/56#discussion_r466514263

Reply:
my thinking rn ^